Formula/godep.rb

```diff
@@ -15,6 +15,8 @@ class Godep < Formula
 
   depends_on "go"
 
+  conflicts_with "deployer", :because => "both install `dep` binaries"
```

Comment:
`godep` doesn't seem to install a `dep` binary?

Reply:
@commitay Fixed, please review.